torch/testing/_internal/codegen/random_topo_test.py

```diff
@@ -325,12 +325,12 @@ def runTest(seed, args):
                 print("jit output: ", jit_oo)
                 print("diff ", jit_oo - oo)
                 raise WrongResultException()
-    except WrongResultException:
+    except WrongResultException as err:
         raise Exception("cuda fuser gives wrong results, repro by running:\n"
-                        "\t{0}".format(reproString(seed, args)))
+                        "\t{0}".format(reproString(seed, args))) from err
     except Exception as err:
         raise Exception("something in cuda fuser went wrong {0}\n\trepro by running:\n\t{1}".format(
-            str(err), reproString(seed, args)))
+            str(err), reproString(seed, args))) from None
```

Comment:
I think chaining `err` would help developer debug the problem

Reply:
Sure. Applied your suggestion (https://github.com/pytorch/pytorch/pull/43836#discussion_r479799938)